Changelog.md

```diff
@@ -0,0 +1,53 @@
+# Changelog
+
+## Version 0.4.0
+
+**In progress**
+
+* [GH73](https://github.com/RoseauTechnologies/Roseau_Load_Flow/issues/73)
+  [PR83](https://github.com/RoseauTechnologies/Roseau_Load_Flow/pull/83) Rename `precision` into `tolerance` and
```

Comment:
The PR number here is wrong. Feel free to release then correct it later, it is not part of the released package so it doesn't matter if it is corrected afterwards.
```suggestion
  [PR85](https://github.com/RoseauTechnologies/Roseau_Load_Flow/pull/85) Rename `precision` into `tolerance` and
```